recon_surf/recon-surf.sh

```diff
@@ -31,6 +31,9 @@ threads="1"           # number of threads to use for running FastSurfer
 allow_root=""         # flag for allowing execution as root user
 atlas3T="false"       # flag to use/do not use the 3t atlas for talairach registration/etiv
 segstats_legacy="false" # flag to enable segstats legacy mode
+base=0                # flag for longitudinal template (base) run
+long=0                # flag for longitudinal time point run
+baseid=""             # baseid for logitudinal time point run
```

Comment:
Is this not redundant information? I think I would have consolidated long and baseid into one variable, because `baseid!=""` is essentially the same as `long==1`

Reply:
I don't really care. I thought about it and found that this way long is more parallel to base (boolean) and think it does not add too much overhead. So I would keep it for now.